backend/scraper/scrape_heb.py

```diff
@@ -1 +1,100 @@
-# TODO scrape items from HEB
+from bs4 import BeautifulSoup
+import requests
+
+# The goal is to have a get_products functions that returns a list of prices given a string query,
+# Parse_link function that gets the name, price,
+# price per unit, and image URL of a specific item. See Ryan's Walmart scraper for examples.
+
+# HEB Scraper Class
+class HEBScraper():
+    def __init__(self):
+        pass

```

Comment:
We'll likely want to handle the location initialization code here. That is, we want prices from the specific HEB located in College Station, not one in like Houston or something. There should be a POST request or something we can use to set that location

Reply:
So like we discussed at the 6/11 meeting, we're going to assume that HEB uses the same prices at all stores, so we can ignore this.